README.md

```diff
@@ -1,38 +1,50 @@
 # Weather Greeting Application
-This project demonstrates how to use Python to create web application. It is a primarily meant for educational purposes, specifically created for the WorldQuant University Introduction to Data Science Module.
+
+This project demonstrates how to use Python to create a web application. It is primarily meant for educational purposes, specifically created for the *WorldQuant University - Introduction to Data Science* Module.
 
 The skills and tools used are:
+
 * Python's `requests` package.
 * Python's `flask` web framework.
-* working with JSONs.
+* working with JSON.
 * modular programming
 
-When a client visits the application, it
+When a client visits the application, it:
+
 1. gets the client's IP address.
-1. uses the IP address to look up their location.
-1. uses their location data to greet them with temperature of the city they are located in.
+2. uses the IP address to look up their location.
```

Comment:
There's no need to make this change. In markdown, numbered lists are created using `1.` but when rendered, the appropriate numbers are used. That way, there's no need to update the numbering when changes are made. A little confusing, I know.

Reply:
Thanks for the tip; this is so much better than manually refreshing numbers. 

I'll restore the 1's I had replaced.